config.guess

```diff
@@ -833,8 +833,11 @@ EOF
 	echo ${UNAME_MACHINE}-pc-minix
 	exit ;;
     arm*:Linux:*:*)
-	echo ${UNAME_MACHINE}-unknown-linux-gnu
-	exit ;;
+  echo ${UNAME_MACHINE}-unknown-linux-gnu
```

Comment:
Tab/spaces formatting issue here, please make sure to use the same indentation characters.

Reply:
thanks, fixed.